spec/rails_helper.rb

```diff
@@ -28,12 +28,16 @@
 #
 # Rails.root.glob('spec/support/**/*.rb').sort.each { |f| require f }
 
-# Checks for pending migrations and applies them before tests are run.
-# If you are not using ActiveRecord, you can remove these lines.
-begin
-  ActiveRecord::Migration.maintain_test_schema!
-rescue ActiveRecord::PendingMigrationError => e
-  abort e.to_s.strip
+if ENV["SMOKE_TEST"] == "true"
+  ActiveRecord::Base.establish_connection adapter: :nulldb
```

Comment:
Can you add a bit more detail in the commit message for why we do this?

Something like:
```
We're using the nulldb gem because our setup requires us to mock a db connection running locally even though we point our capybara instance to a remote server to run these smoke tests.
```